src/views/templates/details/tabs/scripts/components/SysPrepItem/sysprep-utils.ts

```diff
@@ -109,4 +109,4 @@ export const updateSysprepObject = (
   }
 };
 
-export const getSyspredConfigMapName = (volume: V1Volume) => volume?.sysprep?.configMap?.name;
+export const getSysprepConfigMapName = (volume: V1Volume) => volume?.sysprep?.configMap?.name;
```

Comment:
👍🏽 awesome! nice catch!

Reply:
@metalice can you tell me what was the change here?

Reply:
Leon, it is just one character change in the naming: "d" to "p",  _Syspred_ to _Sysprep_ substring in the name of this util.